cpp/daal/src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi_impl.i

```diff
@@ -90,6 +90,10 @@ services::Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::compute(c
     NumericTable * labels = const_cast<NumericTable *>(model->impl()->getLabels().get());
 
     const Parameter * const parameter = static_cast<const Parameter *>(par);
+    DAAL_CHECK(par, services::ErrorNullParameterNotSupported);
+    const bool computeLabels = bool(parameter->resultsToEvaluate & daal::algorithms::classifier::computeClassLabels) && (y != NULL);
+    const bool computeRegressionIndices = bool(parameter->resultsToEvaluate & ResultToComputeId::computeIndicesOfNeighbors) && (outIndices != NULL);
+    const bool computeRegressionDistances = bool(parameter->resultsToEvaluate & ResultToComputeId::computeDistances) && (outDistances != NULL);
```

Comment:
Maybe it makes sense to report error if result is required but result table pointer is nullptr?

Reply:
Okay! I'll fix it